symfony/panther/1.0/manifest.json

```diff
@@ -19,6 +19,20 @@
             "content": "        <extension class=\"Symfony\\Component\\Panther\\ServerExtension\" />",
             "position": "after_target",
             "target": "<extensions>",
+            "warn_if_missing": false
```

Comment:
by renaming the `phpunit.xml.dist` to `phpunit.dist.xml` we also have the advantages that this recipe is compatible so with phpunit 9 and 10